### ISSUE TITLE
De-cluttering the docs

### ISSUE THREAD

**Issue by MaxHillebrand** [zkSNACKs/WasabiDoc#537]
Over the last 30 000 blocks we have written a breathtaking quantity of documentation about Bitcoin privacy and Wasabi. The docs has turned out to be a monumental work of a large scope, as the topic we cover is very broad too.

Naturally, the additions far outweigh the deletions. Consider that initially there was a lot of vuepress code, though eventually most changes are one sentence per line of "code". 
![screenshot_20200206_110956](https://user-images.githubusercontent.com/30683012/73927224-474fd380-48d1-11ea-9551-c6136e7c0318.png)
![screenshot_20200206_111005](https://user-images.githubusercontent.com/30683012/73927233-49b22d80-48d1-11ea-9735-959677e68a6f.png)

There have been plenty of very valuable additions to the docs, however, also many duplicates, too chatty wordings, and redundant information better explained elsewhere. A new user coming to the docs for the first time might be overwhelmed by the insane quantity of knowledge, whereas a power user might want a more dense quality.

As we have discussed in the dev-call, we must start an effort to de-clutter the docs. This issue is here to explain my thoughts on how to approach this, as well as to gain the feedback and suggestions from all Wasabikas who care to contribute.

I suggest the following approach, though this will change with further contemplation and your feedback, so nothing is set in stone.

## Explain how to work with the docs

At the [bottom of the starting page](https://github.com/zkSNACKs/WasabiDoc/blob/master/docs/README.md) is a good place to explain more thoroughly on how to best work through the documentation. This should link a new user to the step by step starting guides, power users to the advanced sections, and emphasize that the search function is very powerful for general inqueries.

Implemented in #538.

## ELI5 introduction

When on the docs starting page, the button `Reclaim Your Privacy` should link to a very easy to understand and quick to read step by step guide on how to install Wasabi, send and receive and coinjoin. This is for a very new user who is here for the first time, it must not overwhelm him, but it should link for each subject to the proper stand alone chapter with a more thorough explanation. This might also be a good place for a info graphic on how to get started.

Implemented in #539.

## Move advanced chapters

In the main chapters of `using-wasabi`, the sidebar is structured by topics, for example ALL installation guides are up top. This includes however the very advanced `Virtual Machines` and `Tails` setup, which is overwhelming the reader. These should be moved down in the sidebar to a separate `Advanced` section.

Further, every chapter should start out with a very simple explain it like I'm 5 guide, then after a horizontal line, explain the advanced nuances of the topic. For example at the top of `Receive.md` should be a trivial explanation of receiving coins, while at the bottom should be nuances of labeling strategies.

Implemented in #541.

## Remove duplicates

There are some chapters, especially in `why-wasabi` that are redundant and explain the same thing over and over again. These should be removed entirely, while the valuable content that is not yet explained elsewhere should be incorporated in the existing works.

## Consistency with FAQ & main chapter

I think a good way to use the FAQ is for a keyword search of the relevant question. It should be a very swift and precise explanation of only one topic. They should further link to the main chapter, where the concept is explained holistically and in context. Here is a duplication, though I think it is valuable to keep it. A one paragraph answer of one specific question, which links to a main chapter where the same question is answered again, but in a much broader scope. For example `FAQ: How to label the address` should explain that it must be `the Observer of the transaction, or who ever knows about the address`, and there should be a link to the `/using-wasabi/Receive.md` chapter where the entire reasoning and tutorial on how to do it is explained in depth.

## Rethinking the pillars

Right now we have three main pillars: `Why Wasabi` [about the reasons for privacy], `Using Wasabi` [about how to use the tool properly], and `Building Wasabi` [about how to contribute and make it better]. There are two additions, the `FAQ` [for an easy to search archive of questions] and the `Glossary` [for an easy to search quick explanation of terms and names].

I personally like this structure, and do not think it is necessary to change. However, there was some conversation on removing the `Building` pillar, as the docs is meant for users, not contributors. I think especially for free and open source projects, the difference between the two groups is very blurry, so the docs should be for both.

Though maybe there is a complete re-thinking of the docs structure in order, I am very open for suggestions. 

## Contribution Game

The past contribution games were a great success, and if there is a zkSNACKs budget for it, as well as a good way to quantifying and qualifying the contributions for de-cluttering, it might be a great idea to have this again.

**Comment by molnard** [comment]
Additional Note:
I would like to propose here to review every docs which linked on wasabiwallet.io.

The first things are the installation guides. The first thing that the user would see is a short-list on how to install Wasabi on the ELI5 level. 

- For Windows the msi file is signed. The Op system will display the developer's signature zkSNACKs Ltd. and verify the validity of that. No need to verify the PGP - it can be removed. 
- For OSX the signing procedure is on its way hopefully in the next release the dmg will be signed. 
- For Linux - PGP required.

**Comment by MaxHillebrand** [comment]
@molnard the PGP verification on windows is removed in #540. I will remove it for macOS too, when the next release has the proper signing too.

**Comment by MaxHillebrand** [comment]
All the chapters in `using-wasabi` under the sub section `Using Wasabi`, [that includes `Wallet Genration` `Receive` `CoinJoin` `Send`] have a `Do ... step by  step` guide at the very beginning of the chapter. I believe this already covers the beginner user who is looking for a tl;dr, as it is at the very top of the page. 

Below follows an in depth analysis of the relevant topics, where more advanced users can get their fix too. For those users who only have a preliminary issue, they can drop off after step by step and will not see these details.

So, I believe this is already a very good setup, and I am not sure how to further de-clutter this, from a structure point of view... 

What do you say, is this good as is, or do we need a over arching change to the structure?

**Comment by MaxHillebrand** [comment]
What shall we do with the pillar on `building-wasabi`?

There was a decision that the very technical documentation shall remain in [the main wallet repo](https://github.com/zkSNACKs/WalletWasabi/tree/master/WalletWasabi.Documentation). Although I think it might be confusing for readers and more difficult to review when having two different documentations, this is the way we move forward - but how?

There are some rather technical chapters that I think can be moved to the main repo
- hardware wallet integration testing
- developer debugging guide
- ports

There are chapters that I think can be removed entirely as it is redundant information
- Technical Overview of Wasabi

But I'm unsure what to do with the other chapters, maybe some can be moved to `why-wasabi`?
- Contribution Checklist
- Contribution Game
- Dojo
- ToDo List
- False positive
- Security policy
- MIT license
- credits

What are your thoughts @yahiheb @davterra @RiccardoMasutti ?

**Comment by molnard** [comment]
According to decluttering efforts, I would move the dev-only contents into the main repository. It is easier to maintain and there you do not have to go into the details and describe everything because it is dev only. 
Also less content in docs will result in easier search experience from the user's perspective. 

Should be moved: 

- https://docs.wasabiwallet.io/building-wasabi/HardwareWalletTestingGuide.html#linux-only-step-0-add-udev-rules
- https://docs.wasabiwallet.io/building-wasabi/HowToDebug.html#before-starting
- https://docs.wasabiwallet.io/building-wasabi/Ports.html
- https://docs.wasabiwallet.io/building-wasabi/FalsePositive.html#email-template

Do not touch: 
- https://docs.wasabiwallet.io/building-wasabi/TechnicalOverview.html#abstract
- https://docs.wasabiwallet.io/building-wasabi/ContributionChecklist.html#who-is-a-contributor
- https://docs.wasabiwallet.io/building-wasabi/ContributionGame.html#contribution-game-of-july-2019
- https://docs.wasabiwallet.io/building-wasabi/Dojo.html#belt-system
- https://docs.wasabiwallet.io/building-wasabi/#contributing-to-wasabi
- https://docs.wasabiwallet.io/building-wasabi/Security.html#security-policy
- https://docs.wasabiwallet.io/building-wasabi/LICENSE.html
- https://docs.wasabiwallet.io/building-wasabi/Credits.html

**Comment by yahiheb** [comment]
Max:
> There are some rather technical chapters that I think can be moved to the main repo
> 
> hardware wallet integration testing
> developer debugging guide
> ports

David:
> Should be moved:
> 
> https://docs.wasabiwallet.io/building-wasabi/HardwareWalletTestingGuide.html#linux-only-step-0-add-udev-rules
> https://docs.wasabiwallet.io/building-wasabi/HowToDebug.html#before-starting
> https://docs.wasabiwallet.io/building-wasabi/Ports.html
> https://docs.wasabiwallet.io/building-wasabi/FalsePositive.html#email-template

I agree with what you both suggest, I am just not sure about the last one that @molnard suggested (FalsePositive document). Should that be moved to the Main repo?

> There are chapters that I think can be removed entirely as it is redundant information
> 
> Technical Overview of Wasabi

There are a lot of valuable information in there so I think it shouldn't be removed.
Maybe just remove the redundant info.

**Comment by MarnixCroes** [comment]
Seems like the mentioned points were tackled.

this can be closed?
